src/app/project/board/board.component.scss

```diff
@@ -40,8 +41,165 @@ section {
 
 // desktop and tablet devices
 @media (min-width: map-get($grid-breakpoints, phone)) {
-   
+
+  .more-space-bottom {
+    margin-bottom: 15px !important;
+  }
+
   .app-toolbar-mobile {
     display: none;
   }
 }
+
+.project-info-container {
+  width: 98vw;
+  max-width: 1280px;
+  position: relative;
+  margin: 0 auto 60px auto;
+
+  .left-info {
+    flex: 70%;
+  }
+
+  .right-info {
+    flex: 30%;
+  }
+
+  .column {
+    padding: 15px;
+  }
+
+  .metadata-box {
+    padding: 19px;
+    margin-bottom: 20px;
+    background-color: whitesmoke;
+    border: 1px solid $primary_100;
+    border-radius: $border-radius;
+    box-shadow: inset 0 1px 1px rgba(0,0,0,.05);
+  }
+
+  .tab-contents {
+    margin-top: 20px;
+  }
+
+  .mat-radio-button {
+    margin-bottom: 5px;
+    display: block;
+  }
+
+  /*
+  .mat-radio-container{
+    height: 15px;
+    width: 15px;
+  }
+  .mat-radio-outer-circle{
+    height: 15px;
+    width: 15px;
+  }
+  .mat-radio-inner-circle{
+    height: 15px;
+    width: 15px;
+  }
+  .mat-radio-button .mat-radio-ripple{
+    height: 15px;
+    width: 15px;
+    left: calc(50% - 15px);
+    top: calc(50% - 15px);
+  }
+   */
+
+  .download-metadata {
+    margin-right: 10px;
+  }
+
+  .sidenav-prop-text {
+    text-align: left;
+    overflow-wrap: break-word;
+    font-weight: inherit;
+    color: #434242;
+  }
+
+  .sidenav-properties-container {
+
+    .property {
+
+      .property-label {
+        font-weight: bold;
+        font-size: 16px;
+        margin-bottom: 8px;
+        text-align: left;
+        overflow-wrap: break-word;
+      }
+
+      .property-value {
+        margin-left: 20px;
+      }
+    }
+
+    .property:not(:last-child) {
+      margin-bottom: 13px;
+    }
+  }
+
+  // grid
+  ::ng-deep .properties-container {
```

Comment:
`::ng-deep` [is deprecated](https://angular.io/guide/component-styles#deprecated-deep--and-ng-deep), however there is still no replacement for this. So we need to keep it in mind to not use it if not necessary.

Reply:
`properties-container` class is used in many children components. So instead of repeating style I have defined the style in parent component with `::ng-deep`. If there is better solution, I am happy to change it.

Reply:
As above, I don't mind to use it, but once they deprecate it for good (it is still marked "to deprecate" in Angular 9), someone needs to fix it.